x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -708,7 +712,7 @@ export class TelemetryReceiver implements ITelemetryReceiver {
     // create and assign an initial point in time
     let pitId: OpenPointInTimeResponse['id'] = (
       await this.esClient.openPointInTime({
-        index: `${this.alertsIndex}*`,
+        index: `${index}*`,
```

Comment:
This is going to break the alerts the recently refactored prebuilt rule alerts. I can push a commit into your branch if you are not sure how to resolve this one.

Reply:
Do you mean [this PR](https://github.com/elastic/kibana/pull/170334/files), right?
Let me fix it and ask you in case I have doubts